### PR TITLE
Update index.html

### LINKED ISSUE #7: Add credit/link to mirror-lake image
Even if we can use the image for free. It is still good to provide some indication of who the owner of the image is.

### LINKED ISSUE #8: Combine `project_description` and `content` blocks in `main.html`
`project_description` and `content` blocks are logically distinct in the context of `index.html` but not in the main layout.

### LINKED ISSUE #9: Be consistent with tabs and spacing
In index.html:
  - some indentations are 2 spaces, others are 4, 12, 16 (?)
  - lots of empty lines

### PR DESCRIPTION
**Pull request by alexandercyu**
Added text to DACCS project description
- added mention of collaboration and the specific agencies and universities
- added more text from proposal to describe goals of the project
- removed mention of "scientific gateway"

2023-11-03
------------
- Reworked opening paragraph into Mission and Vision paragraphs
- Added Funder images
- Merged Affiliation and Expertise for Francis Zweirs and Dan McKenney

2023-11-06
------------
Resolves #7
Resolves #8
Resolves #9

2023-11-07
------------
Made changes from today's meeting
- Moved opening sentence of Mission paragraph to second paragraph
- Added space to:
   - Between DACCS Project title and Mission and Vision sections
   - Above Mission and Vision titles
   - to left and right of Mission and Vision text 
 - Changed Mission and Vision background to white

### CODE REVIEW

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
Please move the funder image updates to a new PR

**Comment by alexandercyu** [comment]
Removed Funder logos for another PR

**Comment by mishaschwartz** [comment]
Please move requirements.txt updates to a new PR

**Comment by mishaschwartz** [comment]
CSS updates should be moved to #10 with the rest of the changes

**Comment by alexandercyu** [comment]
Moved funders css to #10

File: static/css/daccs-project-css.css [hunk 1]
```diff
@@ -4,10 +4,20 @@ body{
...
-.cover-text-box{
+.cover-text-bg{
     background-color: rgba(248, 249, 250, 0.7);
 }
 
+.cover-text-box{
+    background-color: rgba(53, 84, 113, 0.3);
+}
+
+.mission-text-box{
```
Comment (mishaschwartz) [inline static/css/daccs-project-css.css:15]: Why have a different background and border radius for just this box?

File: templates/site/index.html [hunk 2]
```diff
@@ -2,27 +2,53 @@
...
-                 existing algorithms developed by the academic, government and private sectors. Typical outputs are tables, time
-                 series and graphics that support climate change impact studies, vulnerability assessments, and decision-making.
-           </p>
-         </div>
-    </div>
+            <div class="row">
+                <div class="col-5 d-block mx-auto mission-text-box">
+                    <h2>Mission</h2>
+                    <p>
+                        Canada's data infrastructure is inadequate for working with the rapidly growing volume of climate and
```
Comment (mishaschwartz) [inline templates/site/index.html]: This first sentence belongs in the second paragraph. It states the problem that is being solved by the project, this is unrelated to _who_ are the organizations participating in the project

File: templates/site/index.html [hunk 3]
```diff
@@ -2,27 +2,53 @@
...
+                        Earth observation data.  Data Analytics for Canadian Climate Services (DACCS) is a collaboration
+                        between the University of Toronto, the Pacific Climate Impacts Consortium (PCIC), the Computer Research
+                        Institute of Montréal (CRIM), and Ouranos.
+                    </p>
+                    <p>
+                        The Data Analytics for Canadian Climate Services (DACCS) project will simplify access to climate
```
Comment (mishaschwartz) [inline templates/site/index.html]: You already introduced the acronym in the paragraph above, you don't need to spell it out again. Just use "DACCS" here

File: templates/site/index.html [hunk 4]
```diff
@@ -2,27 +2,53 @@
...
+                        analysis tools alongside the data, thus lowering the barrier to innovation in climate science and
+                        opening up new forms of analysis using state-of-the-art machine learning and visualisation tools.
+                    </p>
+                </div>
+                <div class="col-5 d-block mx-auto border rounded cover-text-bg">
+                    <h2>Vision</h2>
+                    <p>
+                        The aim of the DACCS project is to drastically simplify the time-consuming
+                        tasks required to extract relevant climate information from large data sets, allowing researchers to
+                        focus on solving scientific problems rather than on data management issues.
```
Comment (mishaschwartz) [inline templates/site/index.html]: "on data management" ("issues" is not needed)

File: templates/site/index.html [hunk 5]
```diff
@@ -2,27 +2,53 @@
...
-                 community through a unified user interface for: dataset search and discovery; user data management; data analytics;
-                 visualisation; workflow management; publishing; and application deployment. As part of a larger Federated Cloud
-                 Infrastructure, through this gateway, scientists can access data from different sources and process them using
-                 existing algorithms developed by the academic, government and private sectors. Typical outputs are tables, time
-                 series and graphics that support climate change impact studies, vulnerability assessments, and decision-making.
-           </p>
-         </div>
-    </div>
+            <div class="row">
+                <div class="col-5 d-block mx-auto mission-text-box">
```
Comment (mishaschwartz) [inline templates/site/index.html]: I think this looks better as two rows, not side by side columns. But that's just personal preference

File: templates/site/index.html [hunk 6]
```diff
@@ -2,27 +2,53 @@
...
+                </div>
+                <div class="col-5 d-block mx-auto border rounded cover-text-bg">
+                    <h2>Vision</h2>
+                    <p>
+                        The aim of the DACCS project is to drastically simplify the time-consuming
+                        tasks required to extract relevant climate information from large data sets, allowing researchers to
+                        focus on solving scientific problems rather than on data management issues.
+                    </p>
+                    <p>
+                        DACCS is devloping a platform called Marble that will facilitate the conversion of raw climate data
```
Comment (mishaschwartz) [inline templates/site/index.html]: developing

**Review by mishaschwartz — CHANGES_REQUESTED** [review]

File: static/css/daccs-project-css.css (hunk 1 above)
Comment (alexandercyu) [reply static/css/daccs-project-css.css:15]: It's to highlight the Mission statement.  Kind of like this at Ouranos:

https://www.ouranos.ca/en/qui-nous-sommes/mission-vision-et-valeurs

Comment (mishaschwartz) [reply static/css/daccs-project-css.css:15]: Personally I think it looks out of place, especially because most of the text box is empty. But let's ask the rest of the team what they think

File: templates/site/index.html (hunk 6 above)
Comment (alexandercyu) [reply templates/site/index.html]: Fixed spelling

File: templates/site/index.html (hunk 5 above)
Comment (alexandercyu) [reply templates/site/index.html]: I have no preference so, changed to two rows

File: templates/site/index.html (hunk 4 above)
Comment (alexandercyu) [reply templates/site/index.html]: Removed word "issues"

File: templates/site/index.html (hunk 3 above)
Comment (alexandercyu) [reply templates/site/index.html]: Got rid of full use of DACCS

File: templates/site/index.html (hunk 2 above)
Comment (alexandercyu) [reply templates/site/index.html]: Moved sentence to second paragraph

File: static/css/daccs-project-css.css (hunk 1 above)
Comment (alexandercyu) [reply static/css/daccs-project-css.css:15]: Background of text made white as per meeting

**Review by mishaschwartz — APPROVED** [review]
